apps/style/applab/style.scss

```diff
@@ -44,10 +44,11 @@ div#visualizationResizeBar {
   .small-footer-base.responsive {
     max-width: $width;
   }
-  div#codeWorkspace {
+  div.workspace-right {
     left: $width;
   }
-  html[dir='rtl'] div#codeWorkspace {
+  // TODO - should be works-space-right?
```

Comment:
I think the TODO was saying I should change from codeWorkspace to workspace-right, which I've done